tox.ini

```diff
@@ -5,6 +5,24 @@ exclude=*certdata*,*manifestdata*
 jobs=auto
 max-line-length=300
 
-[nosetests]
-with-id=True
-attr='!functional'
+[pytest]
+addopts =
+	# Non-registered markers raise errors
+	--strict-markers
+	# Start with tests that failed last time, --ff
+	--failed-first
+	# Show extra summary for (f)ailed, (E)rror, (s)kipped and (w)arnings
+	-rfEsw
+	# Mark some tests not to be run
+	-m "not zypper and not dbus and not functional"
```

Comment:
with my hints mentioned previously on skipping the zypper test by checking for its required environment variables (and marking it as functional too), there is no more need to exclude it by default, as it will be skipped gracefully